share/github-backup-utils/ghe-gc-disable

```diff
@@ -9,8 +9,8 @@
 set -e
 
 # Bring in the backup configuration
-cd $(dirname "$0")/../..
-. share/github-backup-utils/ghe-backup-config
+export GHE_BACKUP_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
```

Comment:
I would avoid the `export` here to prevent potential regression/behaviour change in a patch release.